### PR TITLE
PDE-3738: Sync docs from v13 zapier-platform release

### PR DESCRIPTION
**Pull request by rnegron**
This PR was created following [this Coda guide](https://coda.io/d/Team-Developer-Platform_di0MgBhlCWf/Syncing-CLI-Docs_su6uK#_luVds) on how to sync docs across repos! 

Here we are updating the docs after the latest v13 release (see PR here: https://github.com/zapier/zapier-platform/pull/611)

### CODE REVIEW

File: docs/_cli_docs/cli.md [hunk 1]
```diff
@@ -64,24 +64,22 @@ This command is typically followed by `zapier upload`.
...
 
-> Convert a Legacy Web Builder app or Visual Builder integration to a CLI integration.
+> Convert a Visual Builder integration to a CLI integration.
 
 **Usage**: `zapier convert INTEGRATIONID PATH`
 
-If you're converting a **Legacy Web Builder** app: the new integration will have a dependency named zapier-platform-legacy-scripting-runner, a shim used to simulate behaviors that are specific to Legacy Web Builder. There could be differences on how the shim simulates and how Legacy Web Builder actually behaves on some edge cases, especially you have custom scripting code.
-
-If you're converting a **Visual Builder** app, then it will be identical and ready to push and use immediately!
+The resulting CLI integraiton will be identical to its Visual Builder version and ready to push and use immediately!
```
Comment (standielpls) [inline docs/_cli_docs/cli.md:71]: `integraiton` spelling

File: docs/_cli_docs/docs.md [hunk 2]
```diff
@@ -3347,6 +3463,4 @@ You can get help by either emailing `partners@zapier.com` or by [joining our dev
 
 ## Developing on the CLI
 
-See [CONTRIBUTING.md](https://github.com/zapier/zapier-platform/blob/master/CONTRIBUTING.md).
-
-{% endraw %}
```
Comment (kreddlear) [inline docs/_cli_docs/docs.md:3352]: Was this purposely removed? If so, we'll likely want to also want to remove the `{% raw %}` at the beginning of the file as well

**Review by kreddlear — CHANGES_REQUESTED** [review]
This looks good aside from one small issue!

File: docs/_cli_docs/docs.md (hunk 2 above)
Comment (standielpls) [inline docs/_cli_docs/docs.md:3352]: I noticed we still have `{% raw %}`, was removing`{% endraw %}` intentional?

File: docs/_cli_docs/docs.md [hunk 3]
```diff
@@ -15,18 +15,138 @@ Zapier is a platform for creating integrations and workflows. This CLI is your g
...
 
 - CLI Docs: [11.3.3](https://github.com/zapier/zapier-platform/blob/zapier-platform-cli@11.3.3/packages/cli/README.md), [10.2.0](https://github.com/zapier/zapier-platform/blob/zapier-platform-cli@10.2.0/packages/cli/README.md), [9.7.3](https://github.com/zapier/zapier-platform/blob/zapier-platform-cli@9.7.3/packages/cli/README.md)
 - CLI Reference: [11.3.3](https://github.com/zapier/zapier-platform/blob/zapier-platform-cli@11.3.3/packages/cli/docs/cli.md), [10.2.0](https://github.com/zapier/zapier-platform/blob/zapier-platform-cli@10.2.0/packages/cli/docs/cli.md), [9.7.3](https://github.com/zapier/zapier-platform/blob/zapier-platform-cli@9.7.3/packages/cli/docs/cli.md)
 - Schema Docs: [11.3.3](https://github.com/zapier/zapier-platform/blob/zapier-platform-schema@11.3.3/packages/schema/docs/build/schema.md), [10.2.0](https://github.com/zapier/zapier-platform/blob/zapier-platform-schema@10.2.0/packages/schema/docs/build/schema.md), [9.7.3](https://github.com/zapier/zapier-platform/blob/zapier-platform-schema@9.7.3/packages/schema/docs/build/schema.md)
 
+## Table of Contents
```
Comment (standielpls) [inline docs/_cli_docs/docs.md:30]: 👍

**Review by standielpls — CHANGES_REQUESTED** [review]
Got tagged to review, looks good! Couple of minor things

File: docs/_cli_docs/docs.md (hunk 2 above)
Comment (standielpls) [reply docs/_cli_docs/docs.md:3352]: resolving - dupe to https://github.com/zapier/visual-builder/pull/401#discussion_r1087125298

File: docs/_cli_docs/cli.md (hunk 1 above)
Comment (rnegron) [reply docs/_cli_docs/cli.md:71]: Shoot; good catch, I'll update this here and open up a PR to fix the source: https://github.com/zapier/zapier-platform/pull/613

**Review by kreddlear — APPROVED** [review]
🎉

**Comment by rnegron** [comment]
@standielpls Applied review suggestions!

**Review by standielpls — APPROVED** [review]

**Pull request merged**